app/uk/gov/hmrc/mobilehelptosave/services/BonusPeriodMilestonesService.scala

```diff
@@ -133,18 +137,18 @@ class HtsBonusPeriodMilestonesService[F[_]](
           StartOfFinalBonusPeriodNoBonus
         )
       )
-    else if (firstPeriodBonusPaid && isAfterFirstPeriod) {
+    else if (firstPeriodBonusPaid && currentBonusTerm != CurrentBonusTerm.First) {
```

Comment:
```suggestion
    else if (firstPeriodBonusPaid && currentBonusTerm == CurrentBonusTerm.Second) {
```
Probably better to use logic currentTerm = Second, otherwise I guess there's a chance this milestone get trigger when the account is closed.

Reply:
So my thinking with this was to cover the (very unlikely) scenario where they don't log in during the whole second term... Would we want that first milestone to never appear?

Reply:
I think its safe to say we probably don't want to show this milestone if they missed the whole second term.

If the above scenario does happen, it would seem a little weird to get a congratz milestone message when the account is close.